src/Extra/Pixlr.php

```diff
@@ -37,7 +39,7 @@ class Pixlr
     protected $secret;
 
     /**
-     * @var MediaManagerInterface
+     * @var ManagerInterface
```

Comment:
Why?

Reply:
Currently `Sonata\MediaBundle\Model\MediaManagerInterface` and `Sonata\CoreBundle\Model\ManagerInterface` deprecated. Changes were not needed?